.github/workflows/build_and_test.yml

```
@@ -69,7 +69,7 @@ jobs:
           colcon-mixin-repository: https://raw.githubusercontent.com/colcon/colcon-mixin-repository/1ddb69bedfd1f04c2f000e95452f7c24a4d6176b/index.yaml
 
       - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v1
+        uses: codecov/codecov-action@v2.0.1
```

```suggestion
        uses: codecov/codecov-action@v2
```
I think `v2` might be reasonable.
https://github.com/codecov/codecov-action/tags